src/formio/components/npFamilyMembers.ts

```diff
@@ -0,0 +1,27 @@
+import {InputComponentSchema, MultipleCapable} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type NpFamilyMembersInputSchema = InputComponentSchema<
+  Record<string, boolean>,
+  Validator,
+  TranslatableKeys
+>;
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+export interface BaseNpFamilyMembersComponentSchema
+  extends Omit<NpFamilyMembersInputSchema, 'hideLabel' | 'disabled'> {
+  type: 'npFamilyMembers';
+  includePartners: boolean;
+  includeChildren: boolean;
+}
+
+/**
+ * @group Form.io components
+ * @category Concrete types
+ */
+export type NpFamilyMembersComponentSchema = MultipleCapable<BaseNpFamilyMembersComponentSchema>;
```

Comment:
`MultipleCapable` is most likely a mistake, I don't think we do anything with this anywhere.

Reply:
![image](https://github.com/open-formulieren/types/assets/65306057/68f9cece-7d81-4002-8654-a28837b2032e)

Thought this meant multipleCapable but turns out it's just the same thing as the select component with multiple values allowed.

Reply:
yeah it kinda does but imo this option doesn't make any sense, since it transforms into `selectboxes` which implicitly is multi capable (otherwise you should use radio)